developer/applications/suitability/index.html

```diff
@@ -0,0 +1,368 @@
+        <label for="commcentersWeight">Community Centers</label>
+        <input id="commcentersWeight" type="range" min="0" max="1" step="any" value="0" />
+        <label for="localparksWeight">Local Parks</label>
+        <input id="localparksWeight" type="range" min="0" max="1" step="any" value="0" />
+      </div>
+      <div id="mapContainer"></div>
+    </div>
+
+    <script type="module">
+      import geojson2h3 from "https://cdn.skypack.dev/geojson2h3@1.0.1";
+
+      // Config
+      const config = {
+        lng: -111.904,
+        lat: 40.719,
+        zoom: 9,
+        fillOpacity: 0.6,
+        colorScale: ["#ffffD9", "#50BAC3", "#1A468A"],
+        h3Resolution: 8
+      };
+
+      // Utilities
+      function normalizeLayer(layer, baseAtZero = false) {
+        const hexagons = Object.keys(layer);
+        // Pass one, get max
+        const max = hexagons.reduce(
+          (max, hex) => Math.max(max, layer[hex]),
+          -Infinity
+        );
+        const min = baseAtZero
+          ? hexagons.reduce((min, hex) => Math.min(min, layer[hex]), Infinity)
+          : 0;
+        // Pass two, normalize
+        hexagons.forEach((hex) => {
+          layer[hex] = (layer[hex] - min) / (max - min);
+        });
+        return layer;
+      }
+
+      function getSliderValue(id) {
+        const input = document.getElementById(id);
+        const value = parseFloat(input.value);
+        console.log(`${id}: ${value}`);
+
+        return value;
+      }
+
+      // Transform a kilometer measurement to a k-ring radius
+      function kmToRadius(km, resolution) {
+        return Math.floor(km / h3.edgeLength(resolution, h3.UNITS.km));
+      }
+
+      function bufferPointsLinear(geojson, radius, h3Resolution) {
+        const layer = {};
+        geojson.features.forEach((feature) => {
+          const [lng, lat] = feature.geometry.coordinates;
+          const stationIndex = h3.geoToH3(
+            lat,
+            lng,
+            h3Resolution
+          );
+          // add surrounding multiple surrounding rings, with less weight in each
+          const rings = h3.kRingDistances(stationIndex, radius);
+          const step = 1 / (radius + 1);
+          rings.forEach((ring, distance) => {
+            ring.forEach((h3Index) => {
+              layer[h3Index] = (layer[h3Index] || 0) + 1 - distance * step;
+            });
+          });
+        });
+
+        return normalizeLayer(layer);
+      }
+
+      let healthdists;
+      let hospitals;
+      let childcare;
+      let grocery;
+      let libraries;
+      let highered;
+      let utabusstops;
+      let utarailstations;
+      let commcenters;
+      let localparks;
+      let charterschools;
+
+      async function getRawData() {
+        healthdists = await d3.json(
+          "https://raw.githubusercontent.com/gregbunce/intergenerational-poverty/main/data/sgid_health_dists_2020.json"
```

Comment:
the reference to healthdists can be removed.  it's not being used in the sliders or anywhere else.